app/templates/components/landing-pro-page.hbs

```diff
@@ -52,6 +52,28 @@
       </section>
       <div class="spacer"></div>
     {{/if}}
+    {{#if this.multiVcs.enableGitlabLogin}}
+      <section class="section-gitlab section--white section--center">
+        <div class="inner">
+          <h2 class="text-5xl text-grey-800 mb-8">
+            Now test your
+            <SvgImage @name='gitlab-full' @width={{220}} />
```

Comment:
Just noticed that there is no `gitlab-full` svg, so no image displays here and it looks odd:
![image](https://user-images.githubusercontent.com/1923979/81079848-124a2180-8ebe-11ea-811d-d79ee0be1b48.png)

Additionally, we may want to consult with Creative and/or Product teams about content for this part. As it is right now it feels a little repetitive. It might be nice to combine the two sections into one somehow, although it would be a little more complex to implement.

Reply:
Nice catch Sophie. We should discuss it with Creative team.

Reply:
Currently, we are going to hide this section due to Marketing decisions.

Reply:
If we do decide to bring it back, I would simplify the copy to this:

### Now test your projects with Travis CI
Check-out our feature - now you can sign up to Travis CI using your Gitlab or Bitbucket account.

Reply:
Hi @sorta , @emmanuel-ci, and @piotr-travisci! Thank you for bringing up the topic! I have a few updates on it:
We are skipping changing the travis-ci.com page in terms of adding GitLab on it. 
It was agreed with the Marketing team that we are going for a silent release for GitLab. It means we need to have signup/in buttons as we have for BB and GB, but we are not mentioning GitLab on the main page.
We are also not doing the separate GitLab landing page as we did for Bitbucket.